Tests/test_PDB_parse_pdb_header.py

```diff
@@ -55,6 +87,16 @@ def test_parse_header_line(self):
         self.assertEqual(header["idcode"], "3EFG")
         self.assertEqual(header["deposition_date"], "2008-09-08")
 
+    def test_parse_title_line(self):
```

Comment:
I'd add a second test with an empty `header['name']` just to make sure it still returns something sensible.

Reply:
done - I think this is your intent, unless you mean adding an additional test file with 'TITLE' in the header but nothing after it.